### PR TITLE
CMR-6849: hopefully the final one. Build should be green now

### PR DESCRIPTION
**Pull request by yiduoke**

### CODE REVIEW

**Review by mitchstartzel — APPROVED** [review]
Approved, assuming the new build is green. Just be sure to start commit messages with `CMR-XXXX: ` in the future, it's tremendously helpful for tracing why changes were made in the future.

File: search-app/src/cmr/search/api/tags.clj [hunk 1]
```diff
@@ -20,10 +20,17 @@
   [headers]
   (mt/extract-header-mime-type #{mt/json} headers "content-type" true))
 
-(defn- tag-api-response
+(defn- has-error?
+  "Whether there's an error in [data]."
+  [data]
+  (some (fn [element] (contains? element :errors)) data))
```
Comment (ygliuvt) [inline search-app/src/cmr/search/api/tags.clj:26]: change this to `(some #(contains? % :errors) data)`

**Review by ygliuvt — APPROVED** [review]
Please use the `squash and merge` option to merge this PR as you have too many small commits and also prefix your commit message with `CMR-6849: ` so that people can reference the commit to the JIRA ticket it is addressing. Also this PR now shows merge conflicts with master and that needs to be addressed as well.

File: search-app/src/cmr/search/api/tags.clj (hunk 1 above)
Comment (yiduoke) [reply search-app/src/cmr/search/api/tags.clj]: resolved

**Pull request merged**